css/index.css

```diff
@@ -0,0 +1,336 @@
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+* {
+  box-sizing: border-box;
+}
+html {
+  font-size: 62.5%;
+}
+html,
+body {
+  font-family: 'Ubuntu', sans-serif;
+  color: #5E6164;
+}
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: 'Russo One', sans-serif;
+  color: #23293B;
+}
+h1 {
+  font-size: 4rem;
+}
+h2 {
+  font-size: 3.2rem;
+}
+h3 {
+  font-size: 2.8rem;
+}
+p {
+  font-size: 1.6rem;
+  line-height: 1.4;
+}
+.container {
+  max-width: 800px;
+  width: 100%;
+  margin: 0 auto;
+}
+@keyframes space {
+  0% {
+    color: #23293B;
+  }
+  25% {
+    color: navy;
+  }
+  50% {
+    color: darkmagenta;
+  }
+  75% {
+    color: purple;
+  }
+  100% {
+    color: lightskyblue;
+  }
+}
+.nav-bar {
+  width: 100%;
+  height: 100px;
+  border-bottom: 2px solid rgba(241, 241, 241, 0.8);
+  background-color: rgba(246, 246, 246, 0.8);
+  padding: 10px 0;
+  position: sticky;
+  top: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.nav-bar h1 {
+  margin-bottom: 15px;
+  animation-name: space;
+  animation-duration: 10s;
+  animation-delay: 2s;
+  animation-iteration-count: 3;
+}
+.nav-bar nav {
+  display: flex;
+  justify-content: space-between;
+  width: 35%;
+}
+.nav-bar nav a {
+  text-decoration: none;
+  font-size: 1.6rem;
+  color: #454A59;
+}
+@media (max-width: 500px) {
+  .nav-bar nav {
+    width: 100%;
+    justify-content: space-around;
+  }
+}
+footer {
+  color: #22283B;
+  background: #f4f4f4;
+  padding: 20px 0;
+  border: 1px double #e0e0e0;
+  border-radius: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.home .intro img {
+  border-radius: 10px;
+  width: 100%;
+}
+.home .intro h2 {
+  padding: 15px;
+}
+.home .intro p {
+  padding: 0 15px;
+}
+.home .space-images {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+.home .space-images img {
+  width: 28%;
+  margin: 20px 2.5%;
+  border-radius: 20px;
+  height: 100%;
+}
+@media (max-width: 500px) {
+  .home .space-images img {
+    width: 100%;
+  }
+}
+@media (max-width: 500px) {
+  .home .space-images .sky {
+    display: none;
+  }
+  .home .space-images .launch {
+    display: none;
+  }
+}
+.home .visit-planets {
+  padding: 15px;
+}
+.home .visit-planets h3 {
+  margin-top: 11px;
+}
+.home .visit-planets p {
+  margin-top: 13px;
+}
+.home .walk {
+  margin-top: 60px;
+  border-top: 1px solid #dee2e6;
+  padding: 50px 60px;
+  display: flex;
+  justify-content: center;
+}
+.home .walk .walk-text {
+  width: 50%;
+}
+.home .walk .walk-text h3 {
+  margin-bottom: 10px;
+}
+.home .walk .walk-text p {
+  margin-bottom: 26px;
+}
+.home .walk .walk-text .btn button {
+  border: 1px solid #22283A;
+  background-color: #22283A;
+  border-radius: 10px;
+  color: #F6F7F8;
+  width: 50%;
+  height: 40px;
+  padding: 10px 0;
+  font-size: 1.7rem;
+  font-weight: bold;
+}
+.home .walk .walk-text .btn button:hover {
+  border: 1px solid #F6F7F8;
+  background-color: #F6F7F8;
+  color: #22283A;
```

Comment:
as we discussed, add a transition : .5s here to give your button a nice fading color effect.